package/android/src/newarch/java/com/reactnativecommunity/slider/ReactSliderManager.java

```diff
@@ -85,16 +83,6 @@ public String getName() {
     return ReactSliderManagerImpl.REACT_CLASS;
   }
 
-  @Override
```

Comment:
Have we got a plan to adapt those Shadow Nodes to new arch https://github.com/facebook/react-native/blob/0.70-stable/ReactCommon/react/renderer/components/slider/SliderShadowNode.cpp ?